python/pyspark/ml/classification.py

```diff
@@ -149,23 +152,23 @@ class ClassificationModel(PredictionModel, _ClassifierParams, metaclass=ABCMeta)
     """
 
     @since("3.0.0")
-    def setRawPredictionCol(self, value):
+    def setRawPredictionCol(self, value: str) -> "P":
         """
         Sets the value of :py:attr:`rawPredictionCol`.
         """
         return self._set(rawPredictionCol=value)
 
     @abstractproperty
```

Comment:
Properties, if followed by another decorator, have to be ignored, i.e.


```python
@abstractproperty   # type: ignore[misc]
@since("3.0.0")
```

Reply:
how is that to ignore? abstractproperty is deprecated also, isn't it?